### DISCUSSION TITLE
Stone chopper shouldn't require fabrication to make.

### DISCUSSION THREAD

**Discussion by BitBite112** [CleverRaven/Cataclysm-DDA#47155] [Ideas]
A stone knife which is made with pressure flaking and costs more resources to make only needs a fabrication level of two, but a stone chopper which should be much easier to make and also only requires rocks requires a fabrication level of 3.

**Comment by anothersimulacrum** [comment]
The stone knife requires survival 5, fabrication 2, the stone chopper only fabrication 3.

**Reply by BitBite112** [reply]
The stone chopper shouldn't require such a high fabrication level. At most it should require a fabrication level of 1. I can't really do an innawoods run without any cutting tools. All the items that let me level up fabrication require cutting tools except for pebbles which go up to level 1.

**Reply by actual-nh** [reply]
Would having the stone chopper require survival 1, fabrication 2 (or similar numbers) better parallel the stone knife? It does seem odd that someone who can make a stone knife might not be able to make a stone chopper.

**Reply by anothersimulacrum** [reply]
I would prefer going the other way, and upping the fabrication requirement for the stone knife to be equal to or greater than that of the stone chopper, unless someone has an argument about how these difficulties do not match reality.

**Reply by BitBite112** [reply]
Making a stone chopper in real life can be as easy as bashing just two rocks together and knocking off a sharp edge off of it. Even a kid could do it just by throwing a rock onto another rock.
In another pr having to do with flint and butchering quality I explained in detail why not all flintknapping is hard to do.
Maybe having the stone chopper requiring fabrication 1 would be more suitable. You'd be able to level it to 1 by making pebbles which is already a thing.
A stone knife would be much more difficult to make because of the method used to make it in real life which would be pressure flaking.

**Comment by danielkwinsor** [comment]
I agree that fabrication should go down on this stone chopper

**Comment by actual-nh** [comment]
Also see #47211.

**Comment by Anaglyphic** [comment]
Why not instead, since it **IS** supposed to be hard to make tools when you don't have any tools, that we add a low-point-cost career option, like (2) "former boy scout master" or something cheese-covered, and he can have a "the scout master manual" book, can will contain all kinds of rudimentary construction recipes for tents and tools and insultingly "indian" ones too... _me smoke-um peacepipe with you, paleface_. This way you can know the recipe without attaining the skill level first, it's stupid anyway with these autolearn recipes that you just inherently know when the time comes... so all this will mean is if you try, a ton of failures, so you'd just collect a _very_ large pile of rocks before starting or get a level or two to mitigate the failure chance?

Or, as part of cabins in woods mapgen, make said recipe book fairly certain drop, or at dropped trailhead signposts, etc. Something lore friendly to enable getting a jumpstart on tool-making without having to follow a strict step-by-step the found online from the wiki or a video for the most efficient way to go from banging rocks together to having a kitted-out forge?
\- \- \-
Also, I hate being the voice of reason - _imagine!_ -  but please be mindful of the "reality vs gameplay" - don't pull those loose threads, mate. You'll just have yarn (20) where your sweater once was.

I mean the "reality" camp is hard to fight against, they are the motivated ones - look at the food system, the caloric density down to the individual calorie, the vitamin load. Firearms rework too, down to accurate magazine capacities, cross-compatible calibres, conversion kits, and proper nomenclature like 'sear' and 'furniture'. _(and the fact whoever put it in, knows the words "sear" and "furniture")_

But then a technician zed can _<yoink!>_ your weapon away, from many tiles distant, due to what, zed magnetism? Stretchy-rubbery limb radiation mutation? Does it matter? It happens because the code says it happens. 

Be mindful of this. I fear the gameplay vs reality is going to suffer. Case in point: Who uses crossbows anymore?

**Reply by actual-nh** [reply]
> Why not instead, since it **IS** supposed to be hard to make tools when you don't have any tools, that we add a low-point-cost career option, like (2) "former boy scout master" or something cheese-covered, and he can have a "the scout master manual" book, can will contain all kinds of rudimentary construction recipes for tents and tools and insultingly "indian" ones too... _me smoke-um peacepipe with you, paleface_. This way you can know the recipe without attaining the skill level first, it's stupid anyway with these autolearn recipes that you just inherently know when the time comes... so all this will mean is if you try, a ton of failures, so you'd just collect a _very_ large pile of rocks before starting or get a level or two to mitigate the failure chance?

Point.

> Or, as part of cabins in woods mapgen, make said recipe book fairly certain drop, or at dropped trailhead signposts, etc. Something lore friendly to enable getting a jumpstart on tool-making without having to follow a strict step-by-step the found online from the wiki or a video for the most efficient way to go from banging rocks together to having a kitted-out forge?

"strict step-by-step the found online from the wiki"?

> But then a technician zed can _<yoink!>_ your weapon away, from many tiles distant, due to what, zed magnetism? Stretchy-rubbery limb radiation mutation? Does it matter? It happens because the code says it happens.

Magnetism. (It could be a tad bit more realistic with regard to what items it would pull, IIRC.)

> Be mindful of this. I fear the gameplay vs reality is going to suffer. Case in point: Who uses crossbows anymore?

Perhaps more emphasis on their being easier to silence, without worrying about subsonic bullets or expending multiple silencers? (And, come to think of it, I suspect that a zed with a longer-than-a-bullet, hard-to-bend/remove (barbs, anyone?) bolt through something critical is going to have a harder time functioning than one with a bullet hole through them. Admittedly, you may find they've incorporated it if you let them revive...)
